docs/ensnode.io/config/integrations/starlight.ts

```diff
@@ -149,7 +150,10 @@ export function starlight(): AstroIntegration {
       replacesTitle: true,
     },
     social: {
+      twitter: "https://x.com/NamehashLabs",
       github: "https://github.com/namehash/ensnode",
+      telegram: "https://t.me/namehash",
```

Comment:
```suggestion
      telegram: "https://t.me/ensnode",
```

We've created a telegram group specifically for ENSNode support.